octobot_trading/errors.py

```diff
@@ -21,6 +21,12 @@ class MissingFunds(Exception):
     """
 
 
+class MinimumOrderCostNotReachedError(Exception):
```

Comment:
👍

Reply:
I don't really understand why we need both `MinimumOrderCostNotReachedError` and `MissingMinimalExchangeTradeVolume`. `MissingMinimalExchangeTradeVolume` is not doing the same thing ? Should we raise it where you are raising `MinimumOrderCostNotReachedError` or you need another exception ?

Reply:
I changed it so its using MissingMinimalExchangeTradeVolume, didnt notice that one